spec/spec_helper.rb

```diff
@@ -6,6 +6,7 @@
 
 require './spec/support/silence_warnings.rb'
 
+require 'logger'
```

Comment:
Rails 6.x tests were failing for a seemingly unrelated reason. I assume some dependency bump broke the build. This fixed it.

Reply:
Sorry I missed it! I wonder if it's worth considering sunsetting rails 6 support in future updates as it's been end of life since October last year. Just a thought to make it a bit easier for future updates, it's not actually that big of an issue at this stage nor does it add too much extra work 🤔